### ISSUE TITLE
Pypi package for 4.6 incomplete

### ISSUE THREAD

**Issue by hmeiland** [devitocodes/devito#1834]
when installing devito==4.6 through pip, the install fails with:
```    
Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/eb-60ept30o/pip-req-build-34mxziu8/setup.py", line 6, in <module>
        with open('requirements.txt') as f:
    FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'
```
When checking the python hosted file (https://files.pythonhosted.org/packages/35/0b/f8439afe6a27114f863dd92c9595bec707f5325556ba3930101fb65dfa6f/devito-4.6.tar.gz) against the github hosted one (https://github.com/devitocodes/devito/archive/refs/tags/v4.6.tar.gz), both the requirements.txt and requirements-optional.txt are indeed missing, while being called in setup.py
```
with open('requirements.txt') as f:
    required = f.read().splitlines()

with open('requirements-optional.txt') as f:
    optionals = f.read().splitlines()
```

**Comment by hmeiland** [comment]
it looks like the requirement installation is not done according to packaging specs; should be something like:
```
install_requires=[
          'numpy',
          'scrapeasy',
      ],
```

**Comment by hmeiland** [comment]
looks like it can be solved by adding
``` include requirements.txt requirements-optional.txt ```
to MANIFEST.in

**Comment by hmeiland** [comment]
should be fixed by https://github.com/devitocodes/devito/pull/1835

**Comment by FabioLuporini** [comment]
closing as fixed by #1835 

Thanks @hmeiland !
